### ISSUE TITLE
Unable to Fetch Occurance List , Program failed with Access Voilation

### ISSUE THREAD

**Issue by pritesha** [ical-org/ical.net#288]
Hi , 
I am trying to use ical.net  to read recurrence rule and parse them to list of occurrences , 
I am using .netcore1.1 and have downloaded version 3.0.8-net-core-beta .

However  it kept on failing if i add any recurrence rule.

Below is my code snippet 👍 
```
 var recurrenceRule = new RecurrencePattern(FrequencyType.Daily, interval: 1)
            {
                Until = DateTime.Parse("2017-06-29T11:59:59")
            };
            var startDateTemp = new DateTime(2017, 05, 30);
            var endDateTemp = new DateTime(2017, 06, 29);

            var vEvent = new CalendarEvent
            {
                DtStart = new CalDateTime(startDateTemp),
                DtEnd = new CalDateTime(endDateTemp),
                RecurrenceRules = new List<RecurrencePattern> { recurrenceRule }
            };

            calendar.Events.Add(vEvent);

            var startDate =  new DateTime(2017, 06, 04);
            var endDate = new DateTime(2017, 06, 20);
            HashSet<Occurrence> occurrences = calendar.GetOccurrences<RecurringComponent>(startDate, endDate);
```

The program exists after the last line with below exception 
`The program '[2072] dotnet.exe' has exited with code -1073741819 (0xc0000005) 'Access violation'.`

did anyone faced this issue before ?  I am fairly new to .net domain .

**Comment by thoemy** [comment]
Should be fixed in 3.0.9-net-core-beta that includes the fix from #282.

**Comment by rianjs** [comment]
Published in Ical.Net 3.0.9-net-core-beta:
https://www.nuget.org/packages/Ical.Net/
